fdbclient/include/fdbclient/TenantManagement.actor.h

```diff
@@ -547,11 +547,11 @@ Future<std::vector<std::pair<TenantName, int64_t>>> listTenants(Reference<DB> db
 }
 
 ACTOR template <class Transaction>
-Future<std::vector<std::pair<TenantName, int64_t>>> listTenantGroupMetadataTransaction(Transaction tr,
-                                                                                       TenantName begin,
-                                                                                       TenantName end,
-                                                                                       int limit,
-                                                                                       StringRef tenantGroup) {
+Future<std::vector<std::pair<TenantName, int64_t>>> listTenantGroupTenantsTransaction(Transaction tr,
```

Comment:
Nit: I would structure this to list the tenant group as the second argument because we are reading from the group rather than filtering based on the group. It's not critical, though, so I'll let you decide.